### ISSUE TITLE
[Bug] Update formula to Support EL8

### ISSUE THREAD

**Issue by ferricoxide** [plus3it/name-computer-formula#15]
Under EL8, The name-computer-formula fails on execution:

~~~
2023-02-21 14:15:15,291 [salt.state       :2192][DEBUG   ][17366] An exception occurred in this state: Failed to restart network.service: Unit network.service not found.
~~~

In digging through logs, it appears that there is an implicit call to restart the `network.service` systemd unit. However, this unit has been deprecated and does not exist on EL8 (or EL9) distros (see [Red Hat Solution #6240271](https://access.redhat.com/solutions/6240271)). Need to track down the implicit call to the the `network.service` systemd unit and replace it with a call to the `NetworkManager.service`.

**Comment by ferricoxide** [comment]
SaltStack project is aware of the isse (see [issue 61538](https://github.com/saltstack/salt/issues/61538)) and issue has been fixed in  3005.1 (see merge-note in [cd9ac8d](https://github.com/saltstack/salt/commit/cd9ac8db9d398b1bd608b7f1af5c0e332eebdc40)).

**Comment by ferricoxide** [comment]
Without embarking on major surgery to work around a vendor-bug that _should_ go away when this project upgrades the saltstack version used, going to close this issue.
